docs/product-installation-guide/src/main/asciidoc/install-dc-standalone-proc.adoc

```diff
@@ -1,21 +1,64 @@
 [id='install-dc-standalone-proc']
 = Installing standalone {CENTRAL} 
 
-You can use the {CENTRAL} standalone JAR file to install {CENTRAL} in a WildFly Swarm environment. This installation method does not require a separate server installation.
+You can use the {CENTRAL} standalone JAR file to run {CENTRAL} without needing to deploy it to an application server, for example {EAP}. 
+[NOTE]
+====
+Red Hat only supports this installation type when it is installed on premise, on Red Hat Enterprise Linux.
+//Any particular version of RHEL?
+====
 
 .Procedure
 
 . Download the {CENTRAL} standalone JAR file from the https://access.redhat.com[Red Hat Customer Portal].
-. Create a `yaml` configuration file as described in http://docs.wildfly-swarm.io/2017.12.1/#configuring-an-application-using-yaml[WildFly Swarm Documentation].
-. In a terminal window, navigate to the directory where you downloaded the installer file and enter the following command:
+. In a terminal window, navigate to the directory where you downloaded the installer file.
+. Create the `application-config.yaml` configuration file with the following contents:
++
+[source]
+----
+swarm:
+  management:
+    security-realms:
+      ApplicationRealm:
+        local-authentication:
+          default-user: local
+          allowed-users: local
+          skip-group-loading: true
+        properties-authentication:
+          path: /path/to/application-users.properties
+          plain-text: true
+        properties-authorization:
+          path: /path/to/application-roles.properties
+datasource:
+  management:
+    wildfly:
+      admin: admin
+----
+. Create the `application-users.properties` file. Include an administrative user and if this {CENTRAL} instance will be a controller for {KIE_SERVER}, include a controller user, for example:
++
+[source]
+----
+rhdmAdmin=password1
+controllerUser=controllerUser1234
+----
++
+. Create the `application-roles.properties` file to assign roles to the users that you included in the `application-users.properties` file, for example:
++
+[source]
+----
+rhdmAdmin=admin
+controllerUser=rest-all
```

Comment:
If I recall correctly the 'rest-all' is no longer valid. You need the role 'kie-server' to make this work. So please change the line `controllerUser=rest-all` to `controllerUser=kie-server`